pmxbot/core.py

```diff
@@ -430,9 +430,9 @@ def merge_dicts(a, b):
 
 
 class Bot(metaclass=abc.ABCMeta):
-	def out(self, channel, s, log=True):
+	def out(self, channel, s, log=True, context=None):
 		try:
-			sent = self.allow(channel, s) and self.transmit(channel, s)
+			sent = self.allow(channel, s) and self.transmit(channel, s, context)
```

Comment:
I would have expected that code to have been caught in the functional tests. Maybe they're being skipped for some reason. I haven't yet had a chance to look.